pom.xml

```
@@ -3,15 +3,20 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.600</version>
+    <version>4.16</version>
   </parent>
 
   <artifactId>join</artifactId>
   <packaging>hpi</packaging>
   <version>1.22-SNAPSHOT</version>
   <name>Join plugin</name>
   <description>Triggering a job after a group of jobs finish</description>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Join+Plugin</url>
+  <url>https://github.com/jenkinsci/join-plugin</url>
+
+  <properties>
+    <jenkins.version>2.222.4</jenkins.version>
```

I've been choosing 2.249.1 as baseline now that 2.277.1 is released.  Should I be preferring 2.222.4 instead?  The [choosing a baseline version page](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) recommends either 2.249.1 or 2.263.1.
```suggestion
    <jenkins.version>2.222.4</jenkins.version>
```

---

I just took the first version with available BOM. No need to alter your approach